src/hints/mod.rs

```diff
@@ -224,7 +224,51 @@ static HINTS: [(&str, HintImpl); 166] = [
     (syscalls::STORAGE_WRITE, syscalls::storage_write),
     (syscalls::SET_SYSCALL_PTR, syscalls::set_syscall_ptr),
     (syscalls::VALIDATE_AND_DISCARD_SYSCALL_PTR, syscalls::validate_and_discard_syscall_ptr),
+
+    (unimplemented::maybe_new::UNUSED_HINT_0, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_1, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_2, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_3, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_4, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_5, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_6, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_7, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_8, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_9, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_10, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_11, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_12, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_13, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_14, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_15, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_16, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_17, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_18, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_19, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_20, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_21, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_22, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_23, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_24, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_25, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_26, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_27, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_28, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_29, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_30, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_31, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_32, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_33, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_34, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_35, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_36, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_37, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_38, hint_stub),
+    (unimplemented::maybe_new::UNUSED_HINT_39, hint_stub),
```

Comment:
I'd prefer if you gave them all a name straight away.

Reply:
8ef3986